cloud/amazon/ec2_elb_lb.py

```diff
@@ -25,6 +25,7 @@
 author:
   - "Jim Dalton (@jsdalton)"
   - "Rick Mendes (@rickmendes)"
+  - "Fernando Jose Pando (@oldmanlinux)"
```

Comment:
I would only do this if you have the blessing of the current authors and you want to take on responsibility for module maintenance